tests/test_mlcontext.py

```diff
@@ -1,39 +1,45 @@
 import pytest
 
 from xdsl.dialects.builtin import UnregisteredAttr, UnregisteredOp
-from xdsl.ir import MLContext, Operation, ParametrizedAttribute, TypeAttribute
-from xdsl.irdl import irdl_attr_definition
+from xdsl.ir import Dialect, MLContext, ParametrizedAttribute, TypeAttribute
+from xdsl.irdl import IRDLOperation, irdl_attr_definition, irdl_op_definition
 
 
-class DummyOp(Operation):
-    name = "dummy"
+@irdl_op_definition
+class DummyOp(IRDLOperation):
+    name = "test.dummy"
 
 
-class DummyOp2(Operation):
-    name = "dummy2"
+@irdl_op_definition
+class DummyOp2(IRDLOperation):
+    name = "test.dummy2"
 
 
 @irdl_attr_definition
 class DummyAttr(ParametrizedAttribute):
-    name = "dummy_attr"
+    name = "test.dummy_attr"
 
 
 @irdl_attr_definition
 class DummyAttr2(ParametrizedAttribute):
-    name = "dummy_attr2"
+    name = "test.dummy_attr2"
+
+
+TestDialect = Dialect("test", [DummyOp], [DummyAttr])
+TestDialect2 = Dialect("test", [DummyOp2], [DummyAttr2])
```

Comment:
I know this is usually the case we use for dialects but it reads a bit weird here. Could you please change them to `TEST_DIALECT` or `testDialect` for these tests?